utils/azureService.js

```diff
@@ -0,0 +1,16 @@
+const { DefaultAzureCredential } = require('@azure/identity');
+const { SecretClient } = require('@azure/keyvault-secrets');
+
+// const keyVaultName = process.env.MONGO_URL;
+const keyValtsUri = `https://horondi.vault.azure.net/secrets`;
```

Comment:
Please, make it an environment variable that is taken from `process.env` and add an example to the `.env.example`

Reply:
fixed